### PR TITLE
Re-Implements a unused variable in NowPlayingWidget.js

### PR DESCRIPTION
**Pull request by SomeRandomOwl**
## About The Pull Request
Re-Implements a unused variable in NowPlayingWidget.js
Now will display the title if there is one in the widget.

## How This Contributes To The Skyrat Roleplay Experience

It will now display any admin played song title above the chat box instead of Now playing: An admin played Soundtrack.

A lot of players frequently ask about the name and link of songs played, this makes it a little more noticeable as to what the name of the song being played is.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

https://files.ratchtnet.com/i/0f58c2

</details>

## Changelog
:cl:
admin: Fixed the Play Internet Sound Variable To now include the song name in the chat box Now Playing Widget.
/:cl:

### CODE REVIEW

**Comment by SomeRandomOwl** [comment]
@Zonespace27 Upstream has this already, SR is the one that originally changed it

**Comment by SomeRandomOwl** [comment]
https://github.com/tgstation/tgstation/blob/master/tgui/packages/tgui-panel/audio/NowPlayingWidget.js

**Comment by Zonespace27** [comment]
"Wait, when the fuck did we change thi-"

> Sees direct commit to master in git blame

"Ahhh."

File: tgui/packages/tgui-panel/audio/NowPlayingWidget.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const NowPlayingWidget = (props, context) => {
               'overflow': 'hidden',
               'text-overflow': 'ellipsis',
             }}>
-            {'An admin-played soundtrack.'}
+            {title || 'An admin-played soundtrack.'}
```
Comment (GoldenAlpharex) [inline tgui/packages/tgui-panel/audio/NowPlayingWidget.js]:
```suggestion
            {title || 'An admin-played soundtrack.'} // SKYRAT EDIT - Replaced 'Unknown Track' with 'An admin-played soundtrack.'
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
This is actually a fix to the code here, as it actually works fine upstream.

**Review by SomeRandomOwl — COMMENTED** [review]
Requested by golden to add a comment to the changed line

File: tgui/packages/tgui-panel/audio/NowPlayingWidget.js [hunk 2]
```diff
@@ -30,7 +30,10 @@ export const NowPlayingWidget = (props, context) => {
               'overflow': 'hidden',
               'text-overflow': 'ellipsis',
             }}>
-            {'An admin-played soundtrack.'}
+            {
```
Comment (Tastyfish) [inline tgui/packages/tgui-panel/audio/NowPlayingWidget.js]: Why aren't we just restoring it to the upstream `{title || 'Unknown Track'}`?

Comment (SomeRandomOwl) [reply tgui/packages/tgui-panel/audio/NowPlayingWidget.js]: Yea that makes the most sense, so i dont have to worry about the linter fighting with me

**Review by Tastyfish — APPROVED** [review]

**Review by GoldenAlpharex — APPROVED** [review]
Fair enough, we can change that string upstream later

**Pull request merged**